src/engine/SCons/cpp.py

```diff
@@ -165,7 +165,7 @@ def CPP_to_Python(s):
     """
     s = CPP_to_Python_Ops_Expression.sub(CPP_to_Python_Ops_Sub, s)
     for expr, repl in CPP_to_Python_Eval_List:
-        s = expr.sub(repl, s)
+        s = re.sub(expr, repl, s)
```

Comment:
This looks wrong? isn't expr a compiled re.. see line 158?

Reply:
Hmmm.  This was in response to the complaint that a string has no `sub` method, but perhaps the code is able to fool pylint.